<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reserveitems</class>
 <widget class="QDialog" name="reserveitems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditItemname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1weekReserve">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>230</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve 1 week</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2weekReserve">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve 2 weeks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3weekReserve">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve 3 weeks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
